public/app/percona/shared/core/reducers/dbaas/addDBCluster/addDBCluster.ts

```diff
@@ -42,7 +44,10 @@ const perconaAddDBClusterSlice = createSlice({
 
 export const addDbClusterAction = createAsyncThunk(
   'percona/addDBCluster',
-  async (args: { values: Record<string, any>; setPMMAddress?: boolean }, thunkAPI): Promise<void> => {
+  async (
+    args: { values: Record<string, any>; setPMMAddress?: boolean; settings?: Settings },
```

Comment:
I think at this point, it might be worth it to create a separate type for the `args`.